site/src/pages/HealthPage/ProvisionerDaemonsPage.tsx

```diff
@@ -115,6 +116,18 @@ export const ProvisionerDaemonsPage: FC = () => {
                     gap: 12,
                   }}
                 >
+                  {
+                    // Add pills for the supported provisioenr types.
+                    daemon.provisioners.map((provType) => {
+                      return (
+                        <Tooltip title="Type" key={daemon.id + provType}>
+                          <Pill icon={<ViewInArIcon />}>
+                            <code>{provType}</code>
```

Comment:
It feels icky to me to allow test code to be bleeding this far into the product UX/UI.  As far as users are/should be concerned, all provisioners are terraform.

Reply:
This is the Health page which is essentially for debugging. I thought I'd add this context. 

I can remove it though, I can just use the api when I want to know.

Reply:
customers can also write their own custom non-tf provisioners, right? could potentially be valuable debugging for them

Reply:
Only if they forked Coder

Reply:
Removed